src/components/AddProposal/AddProposal.component.tsx

```diff
@@ -0,0 +1,186 @@
+      [AddProposalInputs.SendProposalContract]: '1',
+      [AddProposalInputs.Description]: '',
+      [AddProposalInputs.Values]: [
+        {
+          [AddProposalInputs.Target]: '',
+          [AddProposalInputs.Value]: '',
+          [AddProposalInputs.Calidata]: '',
+          [AddProposalInputs.Signature]: '',
+        },
+      ],
+    },
+  });
+
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: AddProposalInputs.Values,
+  });
+
+  return (
+    <AppDialog
+      isOpenDialog={isOpenDialog}
+      onClose={onClose}
+      title="Create Proposal"
+      dialogProps={{
+        scroll: 'body',
+      }}
+      dialogPaperProps={{
+        sx: {
+          width: '800px',
+        },
+      }}
+    >
+      <Box sx={{ width: '100%' }}>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <Box
+            sx={{
+              width: '100%',
+              display: 'grid',
+              gridTemplateColumns: 'repeat(2, 1fr)',
+              gap: 2,
+            }}
+          >
+            <Typography
+              sx={{ alignSelf: 'center', textAlign: 'left' }}
+              variant="h4"
+            >
+              Send Proposal Contract
+            </Typography>
+            <ControlledDropdown
+              name={AddProposalInputs.SendProposalContract}
+              control={control}
+              options={AP_DD_OPTIONS}
+            />
+          </Box>
+
+          <ControlledInput
+            name={AddProposalInputs.Description}
+            control={control}
+            placeholder="Description"
+            multiline
+            rows={4}
+            error={formState.errors[AddProposalInputs.Description]}
+          />
+
+          {fields.map((_, index) => (
+            <Box
+              key={index}
+              sx={({ palette }) => ({
+                p: 1,
+                mt: 1,
+                borderRadius: 1,
+                border: `1px solid ${palette.primary.main}`,
+              })}
+            >
+              <Button
+                variant="text"
+                size="small"
+                onClick={() => {
+                  remove(index);
+                }}
+                sx={{ float: 'right' }}
+              >
+                Delete
+              </Button>
+              <Box
+                sx={() => ({
+                  width: '100%',
+                  display: 'grid',
+                  gridTemplateColumns: 'repeat(2, 1fr)',
+                  gap: 2,
+                })}
+              >
+                <ControlledInput
+                  placeholder="Target"
+                  name={`${AddProposalInputs.Values}.${index}.${AddProposalInputs.Target}`}
+                  control={control}
+                  error={
+                    formState.errors[AddProposalInputs.Values]?.[index]?.[
+                      AddProposalInputs.Target
+                    ]
+                  }
+                />
+                <ControlledInput
```

Comment:
This field should only allow numbers (it's the amount of rBTC). You use the ControlledCurrencyInput that already that logic (I think you can use `RBTC` as the symbol prop)